### PR TITLE
add TTN implementation

### PR DESCRIPTION
**Pull request by DeadByElpy**

### CODE REVIEW

File: src/routes/ttn.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+                        })
+                    });
+                    console.log('send sse to', subscriber.messageId);
+                }
+            }
+        }
+    );
+
+    fastify.post(
+        '/:id/messages',
```
Comment (DarkPark) [inline src/routes/ttn.js:52]:
```suggestion
        '/:eui/messages',
```
?

or change to id in chirpstack

File: src/routes/ttn.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
+// sse listeners
+// {'001a7988170187ca': [{requestId: {request, reply, messageId}}]}
```
Comment (DarkPark) [inline src/routes/ttn.js]: is it a correct example?

**Review by DarkPark — CHANGES_REQUESTED** [review]

File: src/routes/ttn.js (hunk 1 above)
Comment (DeadByElpy) [reply src/routes/ttn.js:52]: changed to `id` in chirpstack

File: src/routes/ttn.js (hunk 2 above)
Comment (DeadByElpy) [reply src/routes/ttn.js]: invalid examples removed

Comment (DarkPark) [reply src/routes/ttn.js]: how about correct the example?

Comment (DarkPark) [reply src/routes/ttn.js]: how about correct the example?

**Pull request merged**